resources/metadata/gitopsUpdateDeployment.yaml

```diff
@@ -65,6 +65,9 @@ spec:
           - name: gitHttpsCredentialsId
             type: secret
             param: username
+          - type: vaultSecret
+            name: gitHttpsCredentialVaultSecretName
+            default: gitHttpsCredential
```

Comment:
Regarding https://github.com/SAP/jenkins-library/issues/3629 (ie. does not yet work in Azure Pipelines): 

Should there be `param: username` here for `vaultSecret` ?  

I don't have prior experience with the Piper task syntax, but array's previous elemenet `gitHttpsCredentialsId` has `param: username` while `gitHttpsCredentialVaultSecretName` only refers to the Vault Secret's KEY NAME and not the payload's attribute name?